cmd/daytona/config/const.go

```diff
@@ -126,6 +126,8 @@ func GetPrebuildScopesFromGitProviderId(providerId string) string {
 		return "admin:repo_hook"
 	case "bitbucket":
 		return "webhooks"
+	case "gitea":
```

Comment:
We can remove this since it's covered by the basic scopes needed to add the git provider.